popup.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-plusplus */
+const projects = ['tonic', 'multi-post stories', 'tonic2', 'mlti-poststories2']; /** name copied */
+const workSection = document.getElementById('portfolio');
+const popup = document.getElementById('popup');
+const closePopUp = document.getElementById('closePopUp');
+const openpop = document.getElementsByClassName('openpop');
+const overlay = document.getElementById('overlay');
+/* const myarticles = document.querySelectorAll('.sections'); */
```

Comment:
- [ ] Please do not leave unused code as comments in your codebase. They make your project look messy. If you need to review a previous version of your code, you can always use git history. 😟